src/components/EventForm/EventForm.tsx

```diff
@@ -45,18 +45,19 @@ function EventForm({ onSubmit, ...rest }: EventFormProps) {
   const [name, setName] = useState(rest.name || '');
   const [imageUrl, setImageUrl] = useState(rest.imageUrl || '');
   const [imageAlt, setImageAlt] = useState(rest.imageAlt || '');
+  const eventDate = rest.startTime || addDays(startOfDay(new Date()), 1);
   const [date, setDate] = useState(
     format(
-      new Date((rest.startTime || new Date(Date.now())).toDateString()) ||
-        Date.now(),
+      eventDate,
       'yyyy-MM-dd'
     )
   );
+
   const [startTime, setStartTime] = useState(
-    format(rest.startTime || Date.now(), 'HH:mm')
+    format(rest.startTime || set(eventDate, { hours: 16, minutes: 0, seconds: 0, milliseconds: 0 }), 'HH:mm')
   );
   const [endTime, setEndTime] = useState(
-    format(rest.endTime || Date.now(), 'HH:mm')
+    format(rest.endTime || set(eventDate, { hours: 17, minutes: 0, seconds: 0, milliseconds: 0 }), 'HH:mm')
```

Comment:
Maybe add a function in `utils/formatting` for formatting time? I just like to define helper functions like that so we avoid repeating the pattern every time and also get to specify the exact locale to use.

Reply:
Created a local function for it